backend/services/implementations/courseModuleService.ts

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable class-methods-use-this */
+import { startSession } from "mongoose";
```

Comment:
using this is so nice!! great idea 😎

Reply:
if we do this here, should we try to include it in every service that also does create and delete?

Reply:
I think we should try to include it in every service that is updating 2 types of data at once to prevent data mismatch 👍